src/components/Profile/components/SchedulePanel/components/myScheduleDialog/index.js

```diff
@@ -15,8 +15,8 @@ import {
   Checkbox,
   FormControl,
   FormControlLabel,
-} from '@material-ui/core';
-import { KeyboardTimePicker } from '@material-ui/pickers';
+} from '@mui/material';
+import { TimePicker } from '@mui/x-date-pickers/TimePicker';
```

Comment:
Yes, it does. The brackets are used to import named, non-default exports. An import without brackets imports the default export, regardless of how it is named.

Reply:
Is `TimePicker` not the default export of `@mui/material/TimePicker?`

Reply:
Looking at the MUI API, it appears that it is, but also that using the default export may have adverse effects on JS bundle-size.

In all the MUI examples, they use named imports. Also, using a named import is nice because it enforces that the imported thing is exported under the same name. So for example, if someone were to change the path of the import without changing the import name, a named import would result in an error, whereas a default import would just create unexpected code.

Reply:
That is nice. I wasn't aware that a default export could affect the bundle size. I feel like we use either or in most cases, so should we default to using names imports wherever possible?

Reply:
I'm not sure it's true that we use either-or. I always use named where possible and encourage others to do so as well.

Reply:
We should have a policy if we know that one thing is better than the other. You just disagreed with me and told me how it was better to do it a certain way.

Reply:
> We should have a policy if we know that one thing is better than the other. You just disagreed with me and told me how it was better to do it a certain way.

I'm not sure what your point is. I believe I said that our policy should be to only use named imports where they're available, because that has already been my policy.

Reply:
I think I misunderstood what you were saying. Yes, sounds good then. We will use the policy that you have been using for named-only imports.

Reply:
Okay, one last question on this. Why wouldn't we leave it as the following? We're doubly specifying that we want to import TimePicker where it could just be:
```suggestion
import { TimePicker } from '@mui/x-date-pickers';
```